3rdparty/smk/smk.h

```diff
@@ -40,7 +40,7 @@ struct Smack {
 	unsigned long m_trackType[7];
 	unsigned long m_extra;
 	unsigned long m_newPalette;
-	unsigned char m_palette[772];
+	unsigned int m_palette[193];
```

Comment:
Is there a reason why this needs changing? Since it's a 3rd party source it probably shouldn't be touched. @Ramen2X

Reply:
Yeah, I agree. Unless there's functionality that actually depends on the member being set this way/and or makes more sense like this, I would leave it how it was. This is how official implementations define this member.